src/component/gamestats/__tests__/__snapshots__/GameStats.test.tsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`GameStats renders the expected game stats 1`] = `
     mistakes: 3
   </div>
   <div>
-    accuracy: 92.7%
+    accuracy: 92.6829268292683%
```

Comment:
Typescript adds this precision?

Reply:
@swcraig fixed - this was a mistake, rounding was removed. [here](https://github.com/daelynj/typinggame/pull/23/commits/332cd9e6287c3987b0b7f6dfff1278cd1fa6d8e9)